### PR TITLE
Added check for PPC PowerNV

### PR DESCRIPTION
**Pull request by shundhammer**
In Ruby, use

    ppc_power_nv?

### CODE REVIEW

File: storage/SystemInfo/Arch.cc [hunk 1]
```diff
@@ -81,6 +81,7 @@ namespace storage
 		y2mil("tmp1:" << tmp1);
 		ppc_mac = boost::starts_with(tmp1, "PowerMac") || boost::starts_with(tmp1, "PowerBook");
 		ppc_pegasos = boost::starts_with(tmp1, "EFIKA5K2");
+                ppc_power_nv = boost::starts_with(tmp1, "PowerNV");
```
Comment (ancorgs) [inline storage/SystemInfo/Arch.cc]: indentation looks strange (here in Github, at least).

Comment (shundhammer) [inline storage/SystemInfo/Arch.cc]: Tabs vs. Blanks issue AFAICS

**Comment by ancorgs** [comment]
A pair of c++-enabled eyes are preferred, but it LGTM.

File: storage/SystemInfo/Arch.h [hunk 2]
```diff
@@ -52,6 +52,7 @@ namespace storage
 
 	bool is_ppc_mac() const { return ppc_mac; }
 	bool is_ppc_pegasos() const { return ppc_pegasos; }
+        bool is_ppc_power_nv() const { return ppc_power_nv; }
```
Comment (k0da) [inline storage/SystemInfo/Arch.h]: is_ppc_powernv ?

Comment (shundhammer) [inline storage/SystemInfo/Arch.h]: Since they normally capitalize the "NV" as in "PowerNV", it's really two words, isn't it?

**Pull request merged**